index.bs

```diff
@@ -857,8 +857,8 @@ nothrow>ReadableStreamAddReadIntoRequest ( <var>stream</var> )</h4>
 
 <emu-alg>
   1. Set _stream_.[[disturbed]] to *true*.
-  1. If _stream_.[[state]] is `"closed"`, return a new promise resolved with *undefined*.
-  1. If _stream_.[[state]] is `"errored"`, return a new promise rejected with _stream_.[[storedError]].
+  1. If _stream_.[[state]] is `"closed"`, return <a>a promise resolved with</a> *undefined*.
+  1. If _stream_.[[state]] is `"errored"`, return <a>a promise rejected with</a> _stream_.[[storedError]].
```

Comment:
"a promise rejected with" seems to be inconsistently linked. I counted 34 places where it is not linked. Am I missing the reason for this?